QUARKUS-6258.md

```diff
@@ -47,6 +54,10 @@ Primary focus will be on RHEL-based testing to ensure functionality of proposed
 
 As we aim for tech preview level for 3.27, we won't be focusing on full platform matrix coverage. We will be gaining experience to expand the coverage and platforms in the upcoming releases.d testing.
 
+#### Impact on test suites for Full Support
+Since Quarkus AI may be moved to full support in its first release (Quarkus 3.27), we need to have a full support on all platforms. Tha means, that all tests should be in Quarkus Test suite, to guarantee running an all supported platforms.
```

Comment:
What is it "full support on all platforms"? RHEL, OpenShift, Windows, IBM Z/PE..? I thought that this is not intended for be RHBQ, hence the platforms may differ to RHBQ. Why are you not more specific, maybe you write it somewhere else? Because in the `Impact on test suite` you mention RHEL and OCP.